### PR TITLE
Refactor API services

### PR DESCRIPTION
**Pull request by jvince**

### CODE REVIEW

File: frontend/src/main.tsx [hunk 1]
```diff
@@ -5,8 +5,8 @@ import App from './components/App';
 import './styles/index.css';
 
 ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
-    <React.StrictMode>
```
Comment (zvonimirr) [inline frontend/src/main.tsx:8]: I'm not a big fan of removing strict mode. It exists for a reason and it's removed automatically in the prod build anyways.

**Review by zvonimirr — CHANGES_REQUESTED** [review]

File: frontend/src/main.tsx (hunk 1 above)
Comment (jvince) [reply frontend/src/main.tsx]: @zvonimirr 

Reverted to using `ReactStrict`
https://github.com/zvonimirr/tenantee/pull/220/commits/b011118799e2296a47dda18882128383bbeaec90

**Review by zvonimirr — APPROVED** [review]
Please add yourself to the CHANGELOG 😃

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 0.4.0
```
Comment (zvonimirr) [inline CHANGELOG.md:3]: You shouldn't create a new changelog when 0.3.0 wasn't even released.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,13 @@
...
+### API
+- Generalized and simplified using Service API's ([@jvince](https://github.com/jvince))
```
Comment (zvonimirr) [inline CHANGELOG.md:5]: This is still related to the UI, please bring it back to the UI section

**Review by zvonimirr — CHANGES_REQUESTED** [review]

**Review by zvonimirr — APPROVED** [review]

**Pull request merged**